io.github.arunsivaramanneo.GPUViewer.yml

```
@@ -34,8 +33,13 @@ modules:
       - install -Dm644 -t /app/share/appdata ${FLATPAK_ID}.appdata.xml
     sources:
       - type: archive
-        url: "https://github.com/arunsivaramanneo/GPU-Viewer/archive/v1.32.tar.gz"
-        sha256: 6e88a0161d51162fc7b8a635a3bccfa8c593ce5a565c0e7acf2e19e1452bc7b4
+        url: https://github.com/arunsivaramanneo/GPU-Viewer/archive/v1.35.tar.gz
+        sha256: dadcd5834855adbc728ed1ef28e009f05b6d9224b0902576356bd503a63b00d4
+        x-checker-data:
+          type: anitya
+          project-id: 235097
+          stable-only: true
+          url-template: https://github.com/arunsivaramanneo/GPU-Viewer/archive/v$version.tar.gz
```

Better use tarball urls with the project names in them, so it's easier to tell what was updated from bots' commit messages.
```suggestion
          url-template: https://github.com/arunsivaramanneo/GPU-Viewer/archive/v$version/GPU-Viewer-$version.tar.gz
```
Same applies to all github-hosted project sources below.

---

Done